### DISCUSSION TITLE
Hare tree-sitter has changed

### DISCUSSION THREAD

**Discussion by folmilsch** [helix-editor/helix#6954] [Troubleshooting]
Dear developers, 

trying to follow the instructions on https://docs.helix-editor.com/install.html to compile the source code on my machine, but it failed with the message that the HARE tree-sitter cannot be accessed via the rev number stated in languages.toml. Indeed, there is a new grammar for the tree-sitter available and when I changed it to the updated version rev, it worked immediately.

Cheers,
Renato

**Comment by the-mikedavis** [comment]
What was the error you saw? We have tree-sitter-hare pinned to `bc26a6a949f2e0d98b7bfc437d459b250900a165` here: https://github.com/helix-editor/helix/blob/c49faf568105f69f2c2725c865e5ae17e3f25092/languages.toml#L1623-L1625

And that revision is in the tree of the tree-sitter-hare git repository: https://git.sr.ht/~ecmma/tree-sitter-hare, so I would be surprised if that grammar fails to fetch.

Some other questions that could help debug this:

* What's your version of `git`?
* Which operating system are you using?
* What's the new grammar you mentioned? Could you link it?
